### PR TITLE
New reuse page

### PR DESCRIPTION
**Pull request by nicolaskempf57**
This requires some changes from https://github.com/etalab/udata-front/pull/181

Close https://github.com/etalab/data.gouv.fr/issues/1044

### CODE REVIEW

**Review by maudetes — COMMENTED** [review]
Review In Progress ;)

File: udata_front/theme/gouvfr/templates/reuse/display.html [hunk 1]
```diff
@@ -36,255 +36,138 @@
...
+                subject-id="{{ reuse.id }}"
+                subject-class="{{ reuse.__class__.__name__ }}"
+            >
             </discussion-threads>
         </div>
     </section>
-    <section class="fr-container fr-py-7w">
-        <h2 id="more-reuses" class="fr-h2">{{ _('More reuses') }}</h2>
-        <p>{{ _('Discover more reuses.') }}</p>
+    <section class="fr-container fr-py-5v">
```
[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/templates/reuse/display.html:168]: Yes ! I'm trying with the same blank state as the one on the dataset page.

What do you think ?

[the remark this answers is not in the document]
Comment (maudetes) [reply udata_front/theme/gouvfr/templates/reuse/display.html:168]: I like the rendering! I think we should add some sections title in any case (found a generic `discussions` title on Figma when there is none)? Else we can't easily understand the different sections I think, ex: http://dev.local:7000/en/reuses/gaillard-2/

[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/templates/reuse/display.html:170]: Xnapper-2023-03-15-09 26 47

Done !

**Review by maudetes — APPROVED** [review]
I think we're all clear! :clap:

**Pull request merged**